soc/arm/st_stm32/stm32g4/soc.c

```diff
@@ -45,6 +50,11 @@ static int stm32g4_init(const struct device *arg)
 	/* allow reflashing board */
 	LL_DBGMCU_EnableDBGSleepMode();
 
+#if defined(PWR_CR3_UCPD_DBDIS)
```

Comment:
ditto

Reply:
yes, 
however, the `stm32g4xx_ll_pwr.h` has this flag  `#if defined(PWR_CR3_UCPD_DBDIS)`